platform/ios/src/MGLMapView.mm

```diff
@@ -1610,19 +1608,16 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
             [MGLMapboxEvents ensureMetricsOptoutExists];
         }
     }
-    else if ([keyPath isEqualToString:@"coordinate"] && [object conformsToProtocol:@protocol(MGLAnnotation)])
+    else if ([keyPath isEqualToString:@"coordinate"] && [object conformsToProtocol:@protocol(MGLAnnotation)] && ![object isKindOfClass:[MGLMultiPoint class]])
     {
         id <MGLAnnotation> annotation = object;
-        MGLAnnotationTag annotationTag = [self annotationTagForAnnotation:annotation];
-        if (annotationTag != MGLAnnotationTagNotFound)
+        MGLAnnotationTag annotationTag = (MGLAnnotationTag)(NSUInteger)context;
```

Comment:
I take it MGLMultiPoint doesn’t return a valid `annotationTag`, which is why we can assume the tag exists (now that we’re no longer trying to support multipoint here)?